articles/connections/social/google.md

```diff
@@ -14,6 +14,8 @@ seo_alias: google
 
 To connect your Auth0 client to Google and Google Apps, you will need to generate a *Client ID* and *Client Secret* in a Google project, copy these keys into your Auth0 settings, and enable the Connection.
 
+If you are looking to manage authentication in your native Android app [see the Android quickstart](/quickstart/native/android).
+
```

Comment:
Yes, agree with Maria. This sentence is not the right way to solve this, we need to address this for all scenarios not just the one with android.